src/sidebar/services/annotations-exporter.ts

```diff
@@ -39,4 +46,50 @@ export class AnnotationsExporter {
       ) as APIAnnotationData[],
     };
   }
+
+  buildTextExportContent(
+    annotations: APIAnnotationData[],
+    groupName = '',
+    /* istanbul ignore next - test seam */
+    now = new Date(),
+  ): string {
+    const [firstAnnotation] = annotations;
+    if (!firstAnnotation) {
+      throw new Error('No annotations to export');
+    }
+
+    const { uri, title } = documentMetadata(firstAnnotation);
+    const uniqueUsers = [
+      ...new Set(
+        annotations
+          .map(annotation => username(annotation.user))
```

Comment:
We should use the user's display name if available. In the LMS context, the usernames are just hashes.